### PR TITLE
Enhancement/digital physical print

### PR DESCRIPTION
**Pull request by chraker**
Improved on  digital print list: 
image

Further added selector and fixed bugs when adding/editing physical print.

### CODE REVIEW

File: frontend/web/src/app/components/digital-print/digital-print-details-panel/digital-print-details-panel.component.ts [hunk 1]
```diff
@@ -1,18 +1,87 @@
...
   }
 
+  ngOnChanges(){
+    if (this.digitalParts && this.table) { // make sure we truly have loaded the parts
+      this.populate();
+    }
+  }
+
+  private populate() {
+    const _self = this;
```
Comment (chraker) [inline frontend/web/src/app/components/digital-print/digital-print-details-panel/digital-print-details-panel.component.ts]: MAJOR Remove this useless assignment to local variable "_self". [rule](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS1854)

**Comment by chraker** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: frontend/web/src/app/components/digital-print/digital-print-details-panel/digital-print-details-panel.component.ts (hunk 1 above)
Comment (chraker) [inline frontend/web/src/app/components/digital-print/digital-print-details-panel/digital-print-details-panel.component.ts]: MAJOR Remove this useless assignment to local variable "_self". [rule](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS1854)

**Comment by chraker** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: frontend/web/src/app/components/digital-print/digital-print-details-panel/digital-print-details-panel.component.ts (hunk 1 above)
Comment (chraker) [inline frontend/web/src/app/components/digital-print/digital-print-details-panel/digital-print-details-panel.component.ts]: MAJOR Remove this useless assignment to local variable "_self". [rule](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS1854)

**Comment by chraker** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: frontend/web/src/app/components/digital-print/digital-print-details-panel/digital-print-details-panel.component.ts (hunk 1 above)
Comment (chraker) [inline frontend/web/src/app/components/digital-print/digital-print-details-panel/digital-print-details-panel.component.ts]: MAJOR Remove this useless assignment to local variable "_self". [rule](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS1854)

**Comment by chraker** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: frontend/web/src/app/components/digital-print/digital-print-details-panel/digital-print-details-panel.component.ts (hunk 1 above)
Comment (chraker) [inline frontend/web/src/app/components/digital-print/digital-print-details-panel/digital-print-details-panel.component.ts]: MAJOR Remove this useless assignment to local variable "_self". [rule](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS1854)

**Comment by chraker** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: frontend/web/src/app/components/digital-print/digital-print-details-panel/digital-print-details-panel.component.ts (hunk 1 above)
Comment (chraker) [inline frontend/web/src/app/components/digital-print/digital-print-details-panel/digital-print-details-panel.component.ts]: MAJOR Remove this useless assignment to local variable "_self". [rule](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS1854)

**Comment by chraker** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: frontend/web/src/app/components/digital-print/digital-print-details-panel/digital-print-details-panel.component.ts (hunk 1 above)
Comment (chraker) [inline frontend/web/src/app/components/digital-print/digital-print-details-panel/digital-print-details-panel.component.ts]: MAJOR Remove this useless assignment to local variable "_self". [rule](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS1854)

**Comment by chraker** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: frontend/web/src/app/components/digital-print/digital-print-details-panel/digital-print-details-panel.component.ts (hunk 1 above)
Comment (chraker) [inline frontend/web/src/app/components/digital-print/digital-print-details-panel/digital-print-details-panel.component.ts:38]: MAJOR Remove this useless assignment to local variable "_self". [rule](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS1854)

**Comment by chraker** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by mlonn — APPROVED** [review]
Bra

**Pull request merged**